### PR TITLE
Fippo selenium all the way

### PR DESCRIPTION
**Pull request by KaptenJansson**
This PR is based on on top of @fippo's work from branch https://github.com/fippo/webrtc/tree/selenium-all-the-way

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,29 @@ language: node_js
...
 
-install:
+env:
+  matrix:
+    - BROWSER=chrome  BVER=stable
+    - BROWSER=chrome  BVER=beta
+    - BROWSER=chrome  BVER=unstable
+    - BROWSER=firefox BVER=stable
+    - BROWSER=firefox BVER=beta
+    - BROWSER=firefox BVER=unstable
```
Comment (fippo) [inline .travis.yml]: add BVER=esr

**Comment by fippo** [comment]
i can add a piece of code that polls for videoWidth every 500ms in a bit.

**Comment by KaptenJansson** [comment]
Firefox ESR does not honor media.navigator.streams.fake = true settings hence why it's failing.

**Comment by jan-ivar** [comment]
@KaptenJansson - Looks like `media.navigator.streams.fake` [was added in 39](https://bugzilla.mozilla.org/show_bug.cgi?id=1110973). Hmm, the workaround would be to modify the constraints. e.g. `{video:true, fake:true}`. Is that doable somehow?

**Comment by fippo** [comment]
@jan-ivar  that explains why it works in adapter -- we're explicitly setting fake:true there. Let's skip ESR for now? Injecting something into localStorage which makes the GUM calls use fake:true might be difficult for any tests that call GUM immediately

**Comment by jan-ivar** [comment]
You mean skip ESR in the samples test? Sure. We'd still test ESR in adapter?

**Comment by KaptenJansson** [comment]
> You mean skip ESR in the samples test? Sure. We'd still test ESR in adapter?
Yes and Yes. We can add it back once it catches up to 39.

**Comment by jan-ivar** [comment]
> We can add it back once it catches up to 39.

Note that the [next ESR](https://www.mozilla.org/en-US/firefox/organizations/faq) will be Firefox 45.

**Comment by jan-ivar** [comment]
Do these tests by any chance run only on linux? If so there may be another way.

**Comment by KaptenJansson** [comment]
Currently only Linux yes but we could if enough people wanted to add the possibility to run on any Windows and Mac as well using the currently installed browsers locally.

**Comment by fippo** [comment]
@tkchin maybe you can help @googlebot with finding the right CLA for my email? The one from https://codereview.webrtc.org/1173403008 should cover that (and github can associate the commits with my github username so...)

**Comment by tkchin** [comment]
AUTHORS doesn't have anything to do with finding CLA. You should check that the git commit has the appropriate email address. I haven't checked, but my guess is that somewhere in the chain of CLs there is an unapproved email address. If you still run into problems @KaptenJansson should be able to  manually override it.

**Comment by KaptenJansson** [comment]
@fippo I just checked and fippo@andynet.net has not been used to sign the CLA while your gmail account has.

**Comment by KaptenJansson** [comment]
But your last PR used fippo@andynet.net and that was approved. This is strange. I think this because I made a PR with your commits, i.e. I'm not the initial author and googlebot@ do not approve this automatically.

**Comment by KaptenJansson** [comment]
Overriding label

**Comment by tkchin** [comment]
Should it be andyet.net?

**Comment by fippo** [comment]
I think googlebot uses my github username usually. The corporate CLA is valid for my andyet.net email but that is not associated with my github username which might be the reason the check fails. But a CLA covering this exists so... (-:

**Comment by samdutton** [comment]
As Zeke suggested, is this a problem with fippo@andy*n*et.net versus
fippo@andyet.net <fippo@andynet.net>?

**Comment by KaptenJansson** [comment]
Also remember you need to sign the CLA per email address you use for the googlebot@ to be happy, at least that's what I did and how I interpret the documentation.

**Pull request merged**